drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -56,13 +56,14 @@
  */
 static void config_bus_clk_init(LL_UTILS_ClkInitTypeDef *clk_init)
 {
+#if defined(CONFIG_SOC_SERIES_STM32WBX) || defined(CONFIG_SOC_STM32WL55XX)
```

Comment:
I do agree that at the current state it is superfluous and that once stm32wl54x is added it has to be extended.
I chose to do it right away with the other fixes such that it is less likely overlooked, and mainly because I'd like to push the config and a board with a stm32wle5x most likely later this week. This way I wouldn't have to take care about clock_control in the new pr.

Could you please confirm that you prefer to leave it with CONFIG_SOC_STM32WL55XX before I make any changes.
Then I'll take care of the different stm32wl socs it in the next PR. Do you think we should introduce a CONFIG_SOC_STM32WL5X and CONFIG_SOC_STM32WLEX in the future to avoid having to check for two socs sperately?

Reply:
@FRASTM I have no changed it exactly as you suggested:
Use CONFIG_SOC_SERIES_STM32WLX for now which later can be changed to CONFIG_SOC_STM32WL5X.

I do have an additional branch that introduces the remaining stm32wlxx socs on top of this and takes care about cpu2 prescaler:
https://github.com/str4t0m/zephyr/tree/stm32wl_socs
How much other stuff do you st guys have to take of in the merge window? Should I open a pr with that branch or would it be only a distraction of more important stuff?

Reply:
OK to "leave it with CONFIG_SOC_STM32WL55XX". Thanks

Reply:
I think that CONFIG_SOC_STM32WLX plus CONFIG_SOC_STM32WL5X (then another  CONFIG_SOC_STM32WLEX when device appears in zephyr) could be fine (later).
@erwango to comment ?

Reply:
Let's keep it that way.
For WL dual core specific code, we'll need to mimic what was done on H7 and add a `STM32WL_DUAL_CORE` symbol.

Reply:
EDIT: We'll need to add a new symbol. I'm not sure `STM32WL_DUAL_CORE` is the exact thing we'll need.
I could also be `STM32WL_M4_CORE`.